### PR TITLE
fix: equality check on tinybird

### PR DESCRIPTION
**Pull request by sshanzel**
Note: do not merge until the other side have been updated to manage the additional column.

### CODE REVIEW

File: src/common/post.ts [hunk 1]
```diff
@@ -202,3 +204,24 @@ export const validatePost = (args: ValidatePostArgs): ValidatePostArgs => {
 
   return { title, content };
 };
+
+export const getPostsTinybirdExport = (con: DataSource, latest: Date) =>
```
Comment (rebelchris) [inline src/common/post.ts:208]: Why did you decide to move this.
Maybe keeping it isolated in tinybird cron is more explicit as to what needs review from Viktor moving forward?

Just to keep the constraint of changes in that file need his approval.

Comment (idoshamun) [reply src/common/post.ts:208]: Agree

**Review by idoshamun — CHANGES_REQUESTED** [review]
Great initiative to add the tests. Let's just accommodate Chris's feedback before

**Review by rebelchris — APPROVED** [review]

**Comment by vpol** [comment]
Could we please keep the order of fields? So in this case can we make `source_type` field go after `post_private`? Thanks

**Comment by rebelchris** [comment]
@vpol Added the latest content_curation change and changed the order.
Please let me know if this is ok for merging

**Review by idoshamun — APPROVED** [review]
Looks good to me. @vpol review and let's get it merged

**Pull request merged**